### ISSUE TITLE
Runtime error caused by `fmt::v6::format_error`

### ISSUE THREAD

**Issue by cubranic** [kotekan/kotekan#517]
There is a `fmt` format error some place, but I can't tell exactly where. It is only triggered occasionally, so it's hard to say exactly what causes it. This is the output when it happens:
```
Buffer visbuf_psr0_5s_3, status: ____________
Buffer visbuf_psr0_5s_merge, status: ____________
terminate called after throwing an instance of 'fmt::v6::format_error'
  what():  invalid type specifier
Aborted
```

**Comment by andrerenard** [comment]
Ah yes, this has gotten me a few times.  I've been using GDB to trace it, but we should make this show up in a cleaner way.   Any ideas @nritsche ?

**Comment by cubranic** [comment]
When I ran it in `gdb`, the message said: "Thread 28 "/dpdk" received signal SIGABRT, Aborted."
So I'd look at logging messages printed by whatever runs in that thread.

Right now, I can't keep kotekan running for more than a few minutes before it terminates with this error.

**Comment by andrerenard** [comment]
@cubranic have you updated your code with the latest develop?  I think I fixed that in #375

**Comment by cubranic** [comment]
I rebased, and that seemed to fix it, thanks.
